Vagrantfile

```diff
@@ -120,6 +120,12 @@ Vagrant.configure('2') do |config|
   Type "vagrant ssh" to login to the shell.
 MSG
 
+  # Temp fix for Vagrant 1.8.5
+  if Vagrant::VERSION =~ /^1.8.5/
```

Comment:
@jvandijk I don't want to force a user to upgrade it's vagrant installation. They might use a different version for another bug or project.